.codesandbox/ci.json

```diff
@@ -11,8 +11,10 @@
     "packages/x-date-pickers",
     "packages/x-date-pickers-pro",
     "packages/x-charts",
+    "packages/x-charts-pro",
```

Comment:
I wanted to check the issue with a CodeSandbox published package and noticed that we are not publishing new pro packages [in CI](https://ci.codesandbox.io/status/mui/mui-x/pr/14335/builds/535143).
Do you think there is a reason for it, or did we just forget to add them to the list? 🤔

Reply:
I think we just forgot it